.npmignore

```diff
@@ -0,0 +1,9 @@
+#for npmignore
+.*
+	.vscode
+	src
+	coverage
+	tests
+	node_modules
```

Comment:
I'm not totally sure whether we should ignore the node_modules folder. I think we should definitely keep the `.npmignore` file around, but maybe keep empty for now. And we'll start adding stuff once we fully know what the minimum required is for the package.